src/explore-education-statistics-frontend/src/components/PageBanner.tsx

```diff
@@ -10,8 +10,14 @@ const PageBanner = () => {
 
         <span className="govuk-phase-banner__text">
           This is a new service – your{' '}
-          <a href="https://www.smartsurvey.co.uk/s/7YS0I/">feedback</a> will
-          help us to improve it.
+          <a
```

Comment:
Looks fine to me, once the first PR with the analytics portion of this PR goes in